### PR TITLE
Heretics changes part 2

### PR DESCRIPTION
**Pull request by whateverusername0**
## About the PR

added ritual of knowledge that required you 3-4 absolutely random items for 4 knowledge points
resets the item pool each successful ritual.

also :cl:

## Why / Balance

alternative to murderboner

🆑 
- add: Added Ritual of Knowledge. Requires you to transmute 4 random items for 4 knowledge points.
- tweak: Successful flesh mark hit will spill the target's entire bloodstream on the ground.
- tweak: Changed void and ash blade sprites.
- tweak: Void blade now deals cold damage.
- tweak: Ash blade now deals heat damage.
- tweak: Seeking blade knowledge now allows void blade to freeze targets.
- tweak: Updated some heretic related sprites

## Summary by CodeRabbit

- **New Features**
	- Introduced temperature manipulation in special effects, enhancing gameplay dynamics.
	- Added a new ritual, "Ritual of Knowledge," expanding gameplay options.
	- Enhanced melee weapon descriptions for better clarity and strategic gameplay.
	- Improved animation timing control with added delay properties for blade states.

- **Bug Fixes**
	- Improved error handling in special effect applications to prevent unintended behavior.

- **Documentation**
	- Updated localization files to include new side quests and clarify existing abilities.

- **Style**
	- Streamlined YAML configurations for better readability and usability. 

- **Chores**
	- Added a dataset for eligible tags to improve item categorization and retrieval.

### CODE REVIEW

File: Resources/Textures/Goobstation/Heretic/Blades/blade_void-inhand.rsi/meta.json [hunk 1]
```diff
@@ -9,11 +9,95 @@
...
+          0.1,
+          0.1
+        ]
+      ]
     },
     {
       "name": "inhand-right",
-      "directions": 4
+      "directions": 4,
+      "delays": [
```
Comment (Piras314) [inline Resources/Textures/Goobstation/Heretic/Blades/blade_void-inhand.rsi/meta.json:59]: I fear no man... but that... thing...... it scares me

File: Resources/Textures/Goobstation/Heretic/Blades/blade_ash.rsi/meta.json [hunk 2]
```diff
@@ -8,7 +8,215 @@
   },
   "states": [
     {
-      "name": "icon"
+      "name": "icon",
+      "delays": [
```
Comment (Piras314) [inline Resources/Textures/Goobstation/Heretic/Blades/blade_ash.rsi/meta.json:12]: there is no way you have to specify every delay if it's all the same please oh my nar'sie save us from these sins

Comment (whateverusername0) [reply Resources/Textures/Goobstation/Heretic/Blades/blade_ash.rsi/meta.json:12]: rsiedit gave me this. it was last updated in june.
we live in a society :finnadie:

File: Resources/Textures/Goobstation/Heretic/Blades/blade_ash-inhand.rsi/meta.json
Comment (Teapug) [inline Resources/Textures/Goobstation/Heretic/Blades/blade_ash-inhand.rsi/meta.json:1]: why oh god oh jesus fucking christ just why


why

Comment (whateverusername0) [reply Resources/Textures/Goobstation/Heretic/Blades/blade_ash-inhand.rsi/meta.json:1]: :finnadie:

**Pull request merged**